.github/workflows/validations.yaml

```
@@ -0,0 +1,39 @@
+name: Validations
+
+on:
+  push:
+    branches:
+      - dev
+      - master  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)